### ISSUE TITLE
Is there a mailing list for this working group?

### ISSUE THREAD

**Issue by afshin** [jupyter/docs-team-compass#35]
Do you have a mailing list for this group? Most other Subprojects and Working Groups use a Google Group mailing list for official communications (election announcements, project-wide communication), for private async discussions, and for allowing the public to contact the group.

I apologize if I have missed it somewhere!

**Comment by chbrandt** [comment]
No, there is not, @afshin . Not yet, will bring into discussion in today's meeting. Thanks for pointing that out.

**Comment by ericsnekbytes** [comment]
Thanks @afshin, we now have a google group, email is [jupyter-docs-working-group@googlegroups.com](mailto:jupyter-docs-working-group@googlegroups.com).

I'll keep this open while we add members and update our docs :)
